rpc/wrtc_signaling_answerer.go

```diff
@@ -306,34 +307,42 @@ func (ans *webrtcSignalingAnswerer) answer(client webrtcpb.SignalingService_Answ
 		var callFlowWG sync.WaitGroup
 		waitOneHost := make(chan struct{})
 		var waitOneHostOnce sync.Once
-		pc.OnICECandidate(func(i *webrtc.ICECandidate) {
+		pc.OnICECandidate(func(icecandidate *webrtc.ICECandidate) {
 			if exchangeCtx.Err() != nil {
 				return
 			}
-			if i != nil {
+			if icecandidate != nil {
 				callFlowWG.Add(1)
-				if i.Typ == webrtc.ICECandidateTypeHost {
+				if icecandidate.Typ == webrtc.ICECandidateTypeHost {
 					waitOneHostOnce.Do(func() {
 						close(waitOneHost)
 					})
 				}
 			}
 			// must spin off to unblock the ICE gatherer
 			utils.PanicCapturingGo(func() {
+				ans.activeBackgroundWorkers.Add(1)
+				defer ans.activeBackgroundWorkers.Done()
+
 				select {
 				case <-initSent:
 				case <-exchangeCtx.Done():
 					return
 				}
-				if i == nil {
+				// there are no more candidates coming during this negotiation
+				if icecandidate == nil {
+					if _, ok := os.LookupEnv("TEST_DELAY_FINISH_NEGOTIATION"); ok {
```

Comment:
I think this is a neat idea making a conditional to replicate a bug based on an environment variable. Hadn't seen that done before. 

Did you intend to leave this in for future developers to be able to reproduce this bug? If so, can you mark this with a comment like:
```
RSDK-4293: Introducing a sleep here replicates the conditions for a prior goroutine leak.
```